### PR TITLE
[PPP-5163] Vulnerable Component: snakeyaml - part 2 ( Non OSGi upgrade)

### PR DESCRIPTION
**Pull request by NJtwentyone**
- change all references to 'org.yaml:snakeyaml'
- add 'org.yaml:snakeyaml' to assembly

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 4m 13s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 53s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>



---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 9m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **1830**, Failures: **0**, Skipped: **17** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1699/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by singletonc — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -2033,7 +2032,7 @@
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>pentaho</groupId>
+        <groupId>pentaho</groupId> <!-- not all dependencies should be excluded, snakeyaml is needed forr KarafBoot.java -->
```
[the remark this answers is not in the document]
Comment (singletonc) [reply pom.xml:2035]: non-blocker, typo in "forr KarafBoot.java"

**Review by rmansoor — APPROVED** [review]

**Pull request merged**